hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3,18 +3,1146 @@
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_mem_encr
+      desc: '''Verify the encryption of the mem within OTBN using the key provided by the OTP.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_idle
+      desc: '''Verify OTBN idle signaling to clkmgr.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    ////////////////////////////////////////////////
+    // Memory & Controllers                       //
+    // ROM, RAM, FLASH, FLASH_CTRL, OTP, OTP_CTRL //
+    ////////////////////////////////////////////////
+
+    // ROM (pre-verified IP) integration tests:
+    {
+      name: chip_rom_access
+      desc: '''Verify access to the rom.
+
+            Verify that the CPU can fetch instructions from the ROM. Nothing extra needs to be done
+            here - all SW tests do this anyway.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_rom_security_features
+      desc: '''Verify ROM security / ECC features if available.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // SRAM (pre-verified IP) integration tests:
+    {
+      name: chip_sram_access
+      desc: '''Verify access to the SRAM.
+
+            Verify that the CPU can fetch data from the SRAM. Nothing extra needs to be done
+            here - all SW tests do this anyway.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sram_encr
+      desc: '''Verify encryption of the SRAM data.
+
+            SW enables encryption within the SRAM. Ensure that the data written to the SRAM is
+            encrypted and likewise, when read from the SRAM is decrypted correctly via backdoor. The
+            key and nonce for the encryption is supplied by the OTP.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sram_ret_access
+      desc: '''Verify access to the SRAM.
+
+            Verify that the CPU can fetch data from the SRAM. Nothing extra needs to be done
+            here - all SW tests do this anyway.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sram_ret_encr
+      desc: '''Verify encryption of the retention SRAM data.
+
+            SW enables encryption within the SRAM. Ensure that the data written to the SRAM is
+            encrypted and likewise, when read from the SRAM is decrypted correctly via backdoor. The
+            key and nonce for the encryption is supplied by the OTP.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sleep_sram_ret_contents
+      desc: '''Verify that the data within the retention SRAM survives low power entry-exit.
+
+            Ensure that the data within the SRAM survives ALL low power entry-exit variations. TODO:
```

Comment:
nit: maybe the TODO should be a newline?

Reply:
SG, thanks.